proprietary/tokens/README.md

```diff
@@ -1,3 +1,229 @@
+You should use these packages if your application uses React, or if it allows applying our HTML classes.
+
+```sh
+npm i @amsterdam/design-system-tokens
+```
+
+```sh
+pnpm add @amsterdam/design-system-tokens
+```
+
+```sh
+yarn add @amsterdam/design-system-tokens
+```
+
+## Usage in CSS
+
+Tokens are typically used as custom properties in CSS.
+
+### Main stylesheet
+
+This package offers a main stylesheet containing all tokens.
+They are declared as global CSS variables through the `:root` selector.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.css";
+```
+
+If that doesn’t work for your project, use `index.theme.css` instead, which uses an `.ams-theme` selector.
+Add that class to a root element of your application.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.theme.css";
+```
+
+```html
+<html class="ams-theme">
+  …
+</html>
+```
+
+### Compact mode
+
+Our branding is rather spacious for websites.
+The main stylesheet implements this ‘spacious mode’ as the default.
+A compact stylesheet is available for applications that require less white space and smaller text.
+
+Note that the compact stylesheet is not independent – it only contains overrides.
+Import it after the main stylesheet for the correct result.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.css";
+import "@amsterdam/design-system-tokens/dist/compact.css";
+```
+
+### Three layers
+
+The tokens are organised in three layers: brand, common and component tokens.
+
+#### Brand tokens
+
+These express the corporate identity of the City of Amsterdam.
+They are our fundamental selection from all possible colours, text characteristics, spacing lengths, border widths, etc.
+Examples:
+
+```css
+:root {
+  --ams-color-primary-red: #ec0000;
+  --ams-space-md: 1rem;
+  --ams-proportion-wide: 4/3;
+  --ams-border-width-lg: 0.1875rem;
+}
+```
+
+Brand tokens must only be referenced by common tokens; not by component tokens or component stylesheets.
+(We are currently working to apply this rule to our own libraries.)
+
+Find the [list of brand tokens](https://github.com/Amsterdam/design-system/tree/main/proprietary/tokens/src/brand/ams) on GitHub.
+
+#### Common tokens
+
+Related components share visual design characteristics.
+E.g. all kinds of links have the same colour, and the borders of various form inputs are equally thick.
+Common tokens express these relations and streamline future changes.
+
+Design system components use common tokens where possible.
+The same goes for custom components that you may create in your application.
+
+```html
+<input class="my-input" type="text" />
+```
+
+```css
+.my-input {
+  font-size: var(--ams-common-input-font-size);
+  border-width: var(--ams-common-input-border-width);
+}
+```
+
+(This is an example; these specific tokens do not exist yet.)
```

Comment:
This is also still confusing imo, why not just use common tokens that exist? A link for example?

Reply:
We can get rid of this line then, no?

```suggestion
```

Reply:
Committed separately, needed to delete an empty line as well to satisfy the linter.